[2025-06-18 15:03:46,276] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2023-01-19T00:00:00+00:00 [queued]>
[2025-06-18 15:03:46,343] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2023-01-19T00:00:00+00:00 [queued]>
[2025-06-18 15:03:46,344] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:03:46,345] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:03:46,345] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:03:46,391] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2023-01-19T00:00:00+00:00
[2025-06-18 15:03:46,402] {standard_task_runner.py:52} INFO - Started process 8373 to run task
[2025-06-18 15:03:46,424] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'second_task', '2023-01-19T00:00:00+00:00', '--job-id', '11277', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp0o6aiurr', '--error-file', '/tmp/tmpv3nhvy2g']
[2025-06-18 15:03:46,434] {standard_task_runner.py:77} INFO - Job 11277: Subtask second_task
[2025-06-18 15:03:46,559] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.second_task 2023-01-19T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:03:46,753] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=my_name
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-19T00:00:00+00:00
[2025-06-18 15:03:46,757] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:03:46,761] {bash.py:158} INFO - Running command: echo "this is sencond task "
[2025-06-18 15:03:46,787] {bash.py:169} INFO - Output:
[2025-06-18 15:03:46,790] {bash.py:173} INFO - this is sencond task
[2025-06-18 15:03:46,791] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:03:46,933] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, execution_date=20230119T000000, start_date=20250618T150346, end_date=20250618T150346
[2025-06-18 15:03:47,072] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:03:47,134] {local_task_job.py:146} INFO - Task exited with return code 0
